.github/workflows/backend.yml

```diff
@@ -127,11 +127,16 @@ jobs:
           # Install `skore` without its dependencies, which are present in the venv
           wheel=(dist/*.whl); python -m pip install --force-reinstall --no-deps "${wheel}"
 
+      - name: Show dependencies versions
```

Comment:
I added this part because it is useful to check the dependency version to have some context.